src/client/modules/my/dataUpload/dataUpload.html

```diff
@@ -0,0 +1,58 @@
+<template>
+
+    <label for="gameId">Game ID:</label>
+    <input type="text" name="gameId">
+
+    <br>
```

Comment:
SLDS Grid probably works better, but if this looks good as-is, I won't complain
https://www.lightningdesignsystem.com/utilities/grid/